executor/distsql.go

```diff
@@ -437,7 +432,7 @@ func (e *IndexLookUpExecutor) startIndexWorker(ctx context.Context, kvRanges []k
 		finished:     e.finished,
 		resultCh:     e.resultCh,
 		keepOrder:    e.keepOrder,
-		batchSize:    e.maxChunkSize,
+		batchSize:    1,
```

Comment:
Why we don't use `initChunkSize`

Reply:
it comes from jira TIDB-2262's discuss using 1 even initChunkSize is other value :laughing: